src/templates/PoolV2/index.tsx

```diff
@@ -33,11 +34,11 @@ import {
 
 import * as S from './styles'
 
-type ListTokensRes = {
+type TokenSwapList = {
```

Comment:
It's an object, it should be named as one, otherwise it does not make sense.

```suggestion
type TokenSwapListItem = {
```

You're using it as `TokenSwapList[]`, reading it out loud, looks like an array of TokenSwapList, like `[TokenSwapList, TokenSwapList, TokenSwapList, ...]`